reference-architecture/aws-ansible/playbooks/roles/etcd-volume/tasks/main.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Create filesystem for /var/lib/etcd
+  filesystem:
+      fstype: xfs
+      dev: /dev/xvdc
+
+- name: Create local volumes directory
+  file:
+      path: /var/lib/etcd
+      state: directory
+      recurse: yes
+
+- name: Create fstab entry
+  mount:
+      name: /var/lib/etcd
+      src: '/dev/xvdc'
+      fstype: xfs
+      state: present
+
+- name: Mount fstab entry
+  mount:
+      name: /var/lib/etcd
+      src: '/dev/xvdc'
+      fstype: xfs
+      state: mounted
```

Comment:
Hmm, odd it looks like the 'disk_setup' module handles it...  I'm not sure why/how it was working previously :)